lua/docs/content/page.tmpl

```diff
@@ -7,6 +7,11 @@
 		<!-- CSS -->
   		<link rel="stylesheet" href="/style/css/style.css">
   		<link rel="stylesheet" href="/style/highlight/atom-one-dark.css">
+		
+		<!-- FONTS -->
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;500;600;700&display=swap" rel="stylesheet">
```

Comment:
you can change the main font in a separate PR, as it will need more discussions with the team